### ISSUE TITLE
combustion's plus critical strike is not displayed in the stats

### ISSUE THREAD

**Issue by wonderchipstick** [ccshiro/cc-buglist#78]
When I put up combustion it's benefit is not visible in the character screen. Also I think the benefit is not granted. With the mage I have ~32% crit chance. The combustion went up to 7 stacks when it wore off. Although it is mathematically possible it is very very unlikely.

**Comment by ccshiro** [comment]
Going to debug it, putting the bug label on for now even though I have not confirmed it yet.

**Comment by ccshiro** [comment]
Debugging shows that the crit chance works as advertised by the spell. Whether or not the character screen should show it I don't know. However, moving that to low priority for now.

Added needs research in case anyone wants to show that it in fact should or shouldn't show up in the character pane. Note that it's a low priority bug, so even with evidence it might take a while until it gets fixed.

**Comment by nz54445** [comment]
Did some research on this, and it's difficult to find multiple credible sources (talk pages, videos etc.). Here is what i found:

Wowwiki:
 Talents that affect the crit chance of "all spells" (i.e.: Conviction), or talents that specifically affect spells of a certain school (i.e.: Critical Mass, Holy Power), directly add to the calculated "Crit Chance %" on the character pane.
Talents that only affect the spell crit chance of a specific spell (such as Incineration and Force of Will) are NOT included in the spell "Crit Chance %" on the character pane. 
http://www.wowwiki.com/Spell_critical_strike

The wowwiki sounds correct. Then i took a look at the mentioned spells on wowwiki (critical mass, holy power):

Critical Mass: Increases the critical strike chance of your fire abilities by 2%/4%/6%. Fairly straightforward, this should reflect fine in the character screen.

Holy Power: Increases the critical effect chance of your holy spells by 1%/2%/3%/4%/5%. Again straightforward.

Finally lets take a look at combustion:

Combustion: When activated, this spell causes each of your fire damage spell hits to increase your critical strike chance with fire damage spells by 10%. This effect lasts until you have caused 3 critical strikes with fire spells.

Note that it specifically states FIRE spells, meaning all of them. By this definition it should indeed appear in the character panel.

Unfortunately, there are no more sources i could find for this specific issue.

tl;dr Combustion effects all fire spells so it should show on the character panel.
